packages/ra-core/src/reducer/admin/references/possibleValues.ts

```diff
@@ -40,8 +40,9 @@ const possibleValuesreducer: Reducer<PossibleValuesState> = (
     }
 };
 
-export const getPossibleReferenceValues = (state, props) =>
-    state[props.referenceSource(props.resource, props.source)];
+export const getPossibleReferenceValues = (state, props) => {
```

Comment:
why that change? extra curly braces bring no more meaning

Reply:
I find it more readable, consistent, and easier to debug.
Sometime  you can endup writing 
```ts
const someFunction = name => firstname => ({ name, firstname });
```
And confuse the function body with another argument.

Whereas with

```ts
const someFunction = name => firstname => {
    return { name, firstname };
};
```
the difference between the arg and the function body is clear.

Also it's easier to add debug this way (typescript refuse the `console.log() ||` trick)
I stopped counting the number of time when I added curly brace just to add debug, end then revert.